packages/chart/src/Column.ts

```diff
@@ -183,7 +183,7 @@ export class Column extends XYAxis {
                     ;
 
                 const domainLength = host.yAxisStacked() ? dataLen : columnScale.bandwidth();
-                columnGEnter.merge(columnGRect as any).each(function (this: SVGElement, d) {
+                columnGEnter.merge(columnGRect as any).each(function (this: SVGElement, d: any) {
```

Comment:
Is this needed on every d3 `.each(` first argument?

Reply:
Just when the compiler complains about it...